transform/mattermost-analytics/models/marts/product/downloads/fct_downloads.sql

```diff
@@ -38,16 +38,15 @@ select
 
 from
     {{ ref('stg_releases__log_entries') }} le
-    left join {{ ref('int_download_stats_per_uri') }} ds on le.uri = ds.uri
 where
     -- Keep only requests with responses at least 1 mb
     response_bytes > 1000000
```

Comment:
I guess this assumes that in the case of a chunked download the minimum chunk size is 1MB(?)

Reply:
TBH this was a filter in the current implementation, so I kept it in order to not have big differences. Let's see the difference between this branch and the current version and we can decide whether to keep it or not.